crates/modelardb_server/src/storage/uncompressed_data_buffer.rs

```diff
@@ -198,14 +199,31 @@ impl fmt::Debug for UncompressedInMemoryDataBuffer {
 
 #[async_trait]
 impl UncompressedDataBuffer for UncompressedInMemoryDataBuffer {
-    /// Finish the array builders and return the data in a structured [`RecordBatch`].
+    /// Finish the array builders and return the data in a [`RecordBatch`] sorted by time.
     async fn record_batch(&mut self) -> Result<RecordBatch, ParquetError> {
-        let timestamps = self.timestamps.finish();
+        // SortColumn requires that the values are in an Arc but this is not needed for take.
```

Comment:
Maybe `but this is not needed for take()` instead to make it clear `take` is a function.

Reply:
Fixed.